### PR TITLE
Add "non-developer" PR category

### PR DESCRIPTION
**Pull request by ChrisBAshton**
These PRs come up quite regularly, and there's always a bit of confusion as to how to handle them.
It will be useful to have a central guide we can point toward next time this comes up.

### CODE REVIEW

File: source/manual/merge-pr.html.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ Some changes require different levels of context and knowledge. For example, a P
 
 If a change is ready to be reviewed but must wait to be merged for some other event, the title should be prefixed with `[Do not merge]`. A description of what the PR is waiting for should be included in the main description of the PR. When a change like this is reviewed, you can simply approve the PR. It's then up to the author to merge that PR when the correct conditions are met.
 
+### A change from a non-developer
+
+Some repositories, such as [smart-answers](https://github.com/alphagov/smart-answers), regularly have PRs opened by content designers, which usually only change text content. As the changes are often made individually through the GitHub UI, they tend to lead to PRs with lots of commits, each changing a single file at a time.
+
+These PRs aren't held to the same commit structure standard as we'd expect of developers. However, if there are a lot of 'fixup' commits, it might be tidier to use the "Squash and merge" option when merging, but making sure to retain any valuable context that may be in the individual commit messages.
```
Comment (kevindew) [inline source/manual/merge-pr.html.md:49]:
```suggestion
These PRs aren't held to the same commit structure standard as we'd expect of developers and can have lots of 'fixup' style commits. As a technologist reviewing these changes you should not expect a non-developer to understand the commit message principles. If you feel the commit messages fall short of GDS standards you should resolve the issues yourself - you can use the "Squash and merge" feature on GitHub (to retain context) or rebase the commits yourself.
```

**Review by kevindew — COMMENTED** [review]
This sounds good, thanks Chris - I've added a rewording suggestion. It might also be nice to have a quick line that says be welcoming and accommodating for a non-developer opening a PR - it can be an intimidating experience.

**Comment by ChrisBAshton** [comment]
Thanks @kevindew. I've rewritten the guidance to factor in your suggestions - could you take a look?
Have also put you down as a co-author!

**Review by kevindew — APPROVED** [review]
Great - thanks @ChrisBAshton

**Pull request merged**